src/components/AddCard/AddCard.jsx

```diff
@@ -0,0 +1,14 @@
+import styles from './AddCard.module.css';
+import Plus from '../../assets/images/plus.svg'
+
+
+function AddCard() {
+  return (
+    <section className={styles.card}>
```

Comment:
Actually, this should be a react-router `Link` tag that points to "/add-application"

Reply:
added the Link tag